lib/node_modules/@stdlib/array/fixed-endian-factory/README.md

```diff
@@ -462,6 +462,38 @@ var str = arr.toString();
 // returns '1,2,3'
 ```
 
+<a name="method-at"></a>
+
+#### TypedArrayFE.prototype.at( i )
```

Comment:
@aayush0325 Would you mind moving this method so that the prototype methods are in alphabetical order?